lib/tasks/db.rake

```diff
@@ -0,0 +1,16 @@
+namespace :db do
+  desc "Add constraint to votes on topics. Vote >= 0"
+  task add_constraints: :environment do
+    ActiveRecord::Base.connection.execute <<-SQL
+      ALTER TABLE topics ADD CONSTRAINT positive_votes CHECK (votes >= 0);
```

Comment:
Woa, that's way cool! I'm surprised that PostgreSQL don't have support for signedness, but those validations are _awesome_! Thanks a lot!